espresso/environment/espresso_dev_node_test.go

```diff
@@ -176,6 +178,83 @@ func runSimpleL2Burn(ctx context.Context, t *testing.T, system *e2esys.System) {
 	cancel()
 }
 
+// TestE2eDevNetWithEspressoSimpleTransactions launches the e2e Dev Net with the Espresso Dev Node
+// and runs a couple of simple transactions to it.
+func TestE2eDevNetWithInvalidAttestation(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	launcher := new(env.EspressoDevNodeLauncherDocker)
+
+	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		t.Fatalf("failed to generate private key")
+	}
+
+	system, _, err :=
+		launcher.StartDevNet(ctx, t,
+			env.SetBatcherKey(*privateKey),
+			env.Config(func(cfg *e2esys.SystemConfig) {
+				cfg.DisableBatcher = true
+			}),
+		)
+
+	if have, want := err, error(nil); have != want {
+		t.Fatalf("failed to start dev environment with espresso dev node:\nhave:\n\t\"%v\"\nwant:\n\t\"%v\"\n", have, want)
+	}
+
+	batchDriver := system.BatchSubmitter.TestDriver()
```

Comment:
So we are not using our own batcher?

Reply:
This is a wrapper over the batcher code, it will use our code internally.

Reply:
Ok cool.